src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1385,7 +1384,7 @@ internal void SaveAs(string path)
         {
             try
             {
-                Model.CurrentWorkspace.SaveAs(path, EngineController.LiveRunnerRuntimeCore);
+                model.SaveAs(path);
```

Comment:
@mjkkirschner Are you thinking more for API user that they should be able to save any workspace when multiple workspace is enabled?